consent-ui/src/app/pis/to-aspsp-page/to-aspsp-page.component.html

```diff
@@ -0,0 +1,8 @@
+<consent-app-to-aspsp
+  [aspspName]="aspspName"
+  [finTechName]="finTechName"
+  [actionName]="'payment'"
```

Comment:
Suggest to have actionName as ENUM or at least string constant reference. It is not dynamic value

Reply:
done